### ISSUE TITLE
Respiratory tract chondritis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11377]
**Preferred term label:**

**Synonyms**
Airway chondritis

**Definition (free text, please give PubMed ID)**
PMID: 24626673 According to Firestein, patients with MAGIC present specific findings for BD and RP, respectively, orogenital aphthous ulcers and chondritis (auricular, nasal, or **respiratory tract**), can also present less specific manifestations such as audiovestibular dysfunction, large artery thrombosis and aneurisms, as other signs and symptoms common to many rheumatic disorders, such as ocular changes, pseudofolliculitis, migratory polyarthritis, glomerulonephritis, gastrointestinal tract involvement, and central nervous system involvement.

PMID: 32914210 Our patient also had features suggesting RP: **respiratory tract chondritis** (hoarseness, shortness of breath), cochlear chondritis (vertigo and tinnitus), and 99mTc-MDP bone scintigraphy suggested costal chondritis.
Moreover, these two clinical entities responded to GC dose increasing and received remission with immunosuppressive agents. Thus, we were able to confirm the diagnosis of MAGIC syndrome in the present case.

**Parent term (use hpo.jax.org/app)**
Chondritis HP:0100662

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MAGIC syndrome (mouth and genital ulcer with inflamed cartilage) ORPHA:324972

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001446
- label: Airway chondritis
- parents: [Chondritis (HP:0100662)](https://hpo.jax.org/app/browse/term/HP:0100662) | [Abnormality of the upper respiratory tract (HP:0002087)](https://hpo.jax.org/app/browse/term/HP:0002087)
- pmid: PMID:40271291
- def: Inflammation of the cartilage of the airway, including especially the trachea and bronchi.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11377
- synonyms: Respiratory tract chondritis(NONE)
